engines/base/src/test/java/org/alfresco/transform/base/clients/HttpClient.java

```diff
@@ -27,8 +28,8 @@
  */
 public class HttpClient
 {
-    private static final RestTemplate REST_TEMPLATE = new RestTemplate();
-    
+    private static final RestTemplate REST_TEMPLATE = MtlsTestUtils.isMtlsEnabled() ? MtlsTestUtils.restTemplateWithMtls() : new RestTemplate();
```

Comment:
I'd move that condition to inside ot MtlsTestUtils.
Call MtlsTestUtils.getRestTemplate() and in that method return either the mtls one or just RestTemplate